CONTRIBUTING.md

```diff
@@ -37,7 +37,9 @@ If you do not own these types of devices, you can emulate them following [these
 
 ### Reporting bug
 
-The first way to contribute to a project is simply reporting a Bug. If you find anything which is not working well or as expected you can [open an issue](https://github.com/decathlon/vitamin-android/issues/new/choose).
+### Reporting an issue
```

Comment:
You need to remove the title before too no? (Reporting bug)

Reply:
well seen 👀